common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +109,7 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            this.setActiveSpeed("1.0"); // default state on initial render
```

Comment:
Why we need this ? It will highlight the wrong value when the speed is not "1.0". You can see in screenshot too.
![screen shot 2016-05-18 at 5 07 51 pm](https://cloud.githubusercontent.com/assets/2994509/15357913/1f23afd4-1d1b-11e6-8761-7da8470cb62d.png)